webdriver/client.py

```diff
@@ -236,11 +237,33 @@ def end(self):
         self.find = None
         self.extension = None
 
+    def send_raw_command(self, method, url, body=None, headers=None):
+        """Send a command to the remote end.
+
+        :param method: HTTP method to use in request
+        :param url: "command part" of the requests URL path
+        :param body: body of the HTTP request
+        :param headers: Additional headers to include in the HTTP request
+        """
+        url = urlparse.urljoin("session/%s/" % self.session_id, url)
+        return self.transport.send(method, url, body, headers)
+
     def send_command(self, method, url, body=None, key=None):
```

Comment:
We should at some point remove `key` from here too, but probably not in this PR.